hw/hw_0001/index.2.js

```diff
@@ -1,21 +1,28 @@
-let goods = [
-    {name: "Beer", cost: 1.5},
-    {name: "Milk", cost: 2.5},
-    {name: "Bread", cost: 4.5},
-    {name: "Butter", cost: 1.5},
-    {name: "Sausages", cost: 3},
+let products = [
+  {name: "Beer", number: "0,5L", price: 17.90},
+  {name: "Butter", number: "400g",price: 30.5},
+  {name: "Sausages", number: "400g",price: 50.3},
+  {name: "Eggs", number: "10 pieces", price: 20.55},
+  {name: "Pepsi", number: "2L", price: 23.9},
+  {name: "Bread", number: "400g", price: 12.8},
+  {name: "Kinkali", number: "0.8kg", price: 39.95},
+  {name: "Milk", number: "0.9kg", price: 38.8},
+  {name: "Wheat flour", number: "2kg", price: 42.95},
+  {name: "Condensed milk", number: "300g", price: 79.35},
+  {name: "White chocolate", number: "100g", price: 74.7},
+  {name: "Sugar", number: "1kg", price: 14.5},
+  {name: "Oatmeal", number: "1kg", price: 23.3},
+  {name: "Marshmallows", number: "500g", price: 45.85},
 ];
 
-var date = new Date();
-
 console.log("My buyings:");
 
-goods.forEach((good) => {
-    console.log(`- ${good.name} costs ${good.cost}`);
+products.forEach((product) => {
+  console.log(`- ${product.name} ${product.number} costs ${product.price}`);
 });
 
-var totalCost = goods.reduce((prev, good) => {
-    return prev + good.cost;
+var totalCost = products.reduce((prev, product) => {
+  return prev + product.price;
```

Comment:
`totalCost` should be calculated as a sum of composition prices and quantities of each product

Reply:
Well, now the program calculates the `total cost` of products from their `weight` and `price` per unit. Also the `total cost` of the number is rounded to two digits after comma